src/components/App.tsx

```diff
@@ -1743,6 +1751,8 @@ export class App extends React.Component<any, AppState> {
     let draggingOccurred = false;
     let hitElement: ExcalidrawElement | null = null;
     let hitElementWasAddedToSelection = false;
+    const elementOriginPosition = { x: 0, y: 0, width: 0, height: 0 };
```

Comment:
I prefer `let elementOriginX` as originally suggested, but this is OK. In this case, we'd want `readonly` keyword in resizeElements.

Reply:
I'll add readonly then 👍